apps/meteor/client/views/room/contextualBar/Info/EditRoomInfo/useEditRoomInitialValues.ts

```diff
@@ -54,7 +58,7 @@ export const useEditRoomInitialValues = (room: IRoomWithRetentionPolicy): Partia
 			hideSysMes: Array.isArray(sysMes) ? !!sysMes?.length : !!sysMes,
 			encrypted,
 			...(canEditRoomRetentionPolicy &&
-				retentionPolicy?.enabled && {
```

Comment:
this needs to be checked in order to be able to display retention policy in rooms